packages/discovery-react-components/src/components/SearchFacets/components/ShowMore/ShowMoreModal.tsx

```diff
@@ -104,10 +129,15 @@ export const ShowMoreModal: FC<ShowMoreModalProps> = ({
       secondaryButtonText={messages.showMoreModalSecondaryButtonText}
       data-testid={`search-facet-show-more-modal-${facetsLabel}`}
     >
+      {filteredFacets && filteredFacets.length == 0 ? (
+        <EmptyModalState messages={messages} />
+      ) : (
+        <></>
+      )}
```

Comment:
How about 
```suggestion
     {filteredFacets && filteredFacets.length === 0 && <p>{messages.emptyModalSearch}</p>}
```
if `EmptyModalState` just renders a single message, I think you don't need that as a separate component.